_includes/topnav.html

```diff
@@ -1,4 +1,106 @@
 <!-- Navigation -->
+<header class="main-header">
+    <div class="container">
+        <div class="main-header__inner">
+            <div class="main-header__left">
+                <strong class="main-header__logo">
+                    <a href="/">
+                        <img src="{{ "/images/logo.svg" }}" alt="Spryker Documentation" class="main-header-logo">
```

Comment:
Why not just `logo` for example? Also we are having `main-header__logo` and `main-header-logo` which is actually confusing a bit.